### DISCUSSION TITLE
Fix for project won't open if closed with no code editors left open!

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#245] [General]
As you folks are probably already aware, there has been an ongoing issue in that if you close a tB project with no code editors open, the next time you open the tB project, it won't open and you have to close VS code and re-open the project to get it to open correctly.

I was just about to start digging into the problem, but I quickly found that it has already been reported to the VS code team, and fixed in the latest builds.  
https://github.com/microsoft/vscode/issues/126302

So if this is affecting you, the fix should be in the next monthly VS Code release in a couple of weeks, **or you can download the Insiders version of VS Code which has the fix included.**  Hooray!

**Comment by Kr00l** [comment]
It seems to be fixed now with v 1.58.0 ?
I do not encounter this issue anymore..

**Reply by WaynePhillipsEA** [reply]
Yes, fixed in the latest VS code release.
